packages/zoe/src/types.js

```diff
@@ -51,9 +51,9 @@
  * @property {MakeInstance} makeInstance
  * @property {Offer} offer
  * @property {(Instance) => Object} getPublicFacet
- * @property {IssuerKeywordRecord} getIssuers
- * @property {BrandKeywordRecord} getBrands
- * @property {Object=} getTerms
+ * @property {(Instance) => IssuerKeywordRecord} getIssuers
+ * @property {(Instance) => BrandKeywordRecord} getBrands
+ * @property {() => Object} getTerms
```

Comment:
I see from the implementation that `getTerms` has an `Instance` parameter as well, as indeed it must.

Reply:
Good point. Fixed